install_guide_pi.md

```diff
@@ -1,14 +1,17 @@
 # Raspbian
 
-Tested with Raspbian Jessie 2017-01-11. 
+RuuviTag is a Bluetooth Low Energy device. 
+Bluetooth 4.0 support is required from the Bluetooth adapter. Raspberry Pi 3 as well as Pi Zero W have integrated Bluetooth which support BLE devices. For older models a Bluetooth adapter is required.
 
-RuuviTag is a Bluetooth Low Energy device, so Bluetooth 4.0 support is required from the Bluetooth adapter. Raspberry Pi 3 has integrated Bluetooth which support BLE devices. For older models Bluetooth adapter is required.
+Tested with Raspbian Jessie 2017-01-11 and '4.9.35-v7+ #1014 SMP Fri Jun 30 14:47:43 BST 2017'
 
-### Start
+<!--  I think it is reasonable to expect the user has a pi up and running
```

Comment:
This is a good point, but let's keep it there. Pretty often I have to use my own guides and I find it nice that they have instructions starting from the basics, so I can get things up and running fast.